src/components/Table/helpers/SortIcon/sortHelpers.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
What's the difference between this and `sortIcon.tsx`?

Reply:
renamed the drectory and files to `SortIcon`